File: dev_stt_2019/embedded_system_surprisy/embedded_system/library_cpp/led_cpp/test_led.cpp
#include "Led.h"            //Here. Again player.h must be in the current directory. or use relative or absolute path to it.
#include "Led.cpp"
int main(){
    Led red(18,"Rosso");
    Led yellow(13,"Giallo");
    Led green(4,"Verde");
    Led blue(17,"Blu");


    red.on();
    green.off();
    yellow.off();
    blue.off();
    delay(1000);			     // wait 1 sec

    red.off();
    green.on();
    yellow.off();
    blue.off();
    delay(1000);			     // wait 1 sec

    red.off();
    green.off();
    yellow.on();
    blue.off();
    delay(1000);			     // wait 1 sec

    red.off();
    green.off();
    yellow.off();
    blue.on();
    delay(1000);			     // wait 1 sec

    red.off();
    green.off();
    yellow.off();
    blue.off();

    return 0;
}

File: dev_stt_2019/embedded_system_surprisy/embedded_system/library_py/led_py/test_led.py
from Led import Led
import time

# button = Led(13); #Yellow
# button = Led(4); #GREEN
# button = Led(18); #Red
button = Led(17); #BLUE
while True:
    # print("ciao");
    button.on();
    time.sleep(0.1)
    button.off();
    time.sleep(0.1)
